BaaServiceKitTests/Mocks/Error+BaaServiceKitTests.swift

```diff
@@ -0,0 +1,3 @@
+enum E: Swift.Error {
```

Comment:
I think `E`enum name is not so talkative. Should use `MockingError` or `TestingError` or something like this.

Reply:
`MockingError` sounds nice